### DISCUSSION TITLE
Snippets and snippet file

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#35] [Q&A]
What is the significance of the snippet file selection below the snippet input?
I can't save anything there right?
![grafik](https://github.com/user-attachments/assets/76d83e4f-09da-4bd1-965a-488e2e4f1016)

**Comment by marcantondahmen** [comment]
No, but you can select snippet files there.

**Comment by cracksilver** [comment]
But how to save the snippets in a new file? 
Manually with external editor, save as php or html in /packages/automad/revitron/templates/snippets ?

**Comment by marcantondahmen** [comment]
Snippets can be saved as `.php` files inside a `snippets` directory inside a package.
